servers/rendering/renderer_scene_cull.cpp

```diff
@@ -1497,7 +1497,15 @@ void RendererSceneCull::instance_geometry_set_shader_parameter(RID p_instance, c
 		isp.value = p_value;
 		instance->instance_shader_uniforms[p_parameter] = isp;
 	} else {
-		E->value.value = p_value;
```

Comment:
Update:

I noticed that my pr only works if an debugger attached, or be more precisely, only after the `instance->instance_shader_uniforms` is correctly inited by other code, or it will go to the `if(!E)` branch, it doesn't feels right because in that branch the type info is lost.